lightningd/bitcoind.c

```diff
@@ -716,6 +716,23 @@ static const char **cmdarr(const tal_t *ctx, const struct bitcoind *bitcoind,
 	return args;
 }
 
+void report_bitcoind_unavailability_and_exit(struct bitcoind *bitcoind, const char *error_message)
```

Comment:
Quite an unwieldy name :wink:

Reply:
Agreed! Any suggestions on naming? :-)

Reply:
`fatal_bitcoind_failure`? It'd match our existing `fatal(msg)` functions.